typings/vscode-mssql.d.ts

```diff
@@ -499,10 +499,10 @@ declare module 'vscode-mssql' {
 	}
 
 	/**
-	* Interface containing deployment options of integer type, value property holds values from <DacFx>\Product\Source\DeploymentApi\ObjectTypes.cs enum
+	* Interface containing deployment options of string[] type, value property holds enum names(nothing but option name) from <DacFx>\Product\Source\DeploymentApi\ObjectTypes.cs enum
 	*/
-	export interface DacDeployOptionPropertyObject {
-		value: number[];
+	export interface DacDeployOptionPropertyStringArray {
```

Comment:
why adding the type 'string' in the name? you don't have to rename the interface name

Reply:
This interface was created for excludeobjects types only, I used the object and booleans to represent the data types they are holding. So, modified the name to the value datatype!

Reply:
reverted to DacDeployOptionPropertyObject! Thanks